packages/gafl-webapp-service/src/pages/summary/licence-summary/__tests__/route.spec.js

```diff
@@ -25,6 +26,21 @@ describe('licence-summary > route', () => {
     })
   })
 
+  describe('checkNavigation', () => {
+    const checkNavigationRedirectError = new GetDataRedirect(CONTACT.uri)
+    it('should return a redirect error if firstName is not included on the licensee', () => {
+      const permission = { licensee: {} }
+      const func = () => checkNavigation(permission)
+      expect(func).toThrow(checkNavigationRedirectError)
```

Comment:
```suggestion
      expect(func).rejects.toThrow(checkNavigationRedirectError)
```
this doesn't fail if you don't throw the error - I think it needs the `.reject`